scripts/o3de/tests/test_engine_template.py

```diff
@@ -297,6 +342,70 @@ def instantiate_template_wrapper(self, tmpdir, create_from_template_func, instan
 
             return test_folder
 
+    #test create Remote repo
+    @pytest.mark.parametrize(
+        "force, expect_results",
+         [
+            pytest.param(True, 0),
+            pytest.param(False, 0)
+        ]
+    )
+    def test_create_repo(self, tmpdir, force, expect_results):
+        concrete_contents = TEST_REPO_JSON
+        templated_contents = TEST_TEMPLATE_REPO_JSON 
+        template_json_contents = TEST_TEMPLATE_REMOTEREPO_JSON_CONTENTS
+
+        # Append the project.json to the list of files to copy from the template
+        template_json_dict = json.loads(template_json_contents)
+        template_json_dict.setdefault('copyFiles', []).append(
+        {
+            "file": "repo.json",
+            "isTemplated": True
+        })
+
+        # Convert the python dictionary back into a json string
+        template_json_contents = json.dumps(template_json_dict, indent=4)
```

Comment:
`repo.json` copyFiles section is already part of `TEST_TEMPLATE_REMOTEREPO_JSON_CONTENTS`
```suggestion
```